### PR TITLE
Readme badges

### PR DESCRIPTION
**Pull request by hlydecker**
I was working on putting together some pretty github documentation for a different project, and I came across how to make these little badges. IDK if this is of any use, but here are three fairly simple examples.

### CODE REVIEW

**Comment by jnothman** [comment]
Thanks! Why not use markdown syntax?

**Comment by hlydecker** [comment]
> Thanks! Why not use markdown syntax?

Verbatim reverse engineered from the Shields readme page :P

File: README.md [hunk 1]
```diff
@@ -1,5 +1,11 @@
 # Welcome to Weed-AI
 
+
+[<img src="https://img.shields.io/github/contributors/Sydney-Informatics-Hub/Weed-AI">](https://github.com/Sydney-Informatics-Hub/Weed-AI/graphs/contributors)
```
Comment (jnothman) [inline README.md:4]:
```suggestion
[![Contributors](https://img.shields.io/github/contributors/Sydney-Informatics-Hub/Weed-AI)](https://github.com/Sydney-Informatics-Hub/Weed-AI/graphs/contributors)
```

**Review by jnothman — APPROVED** [review]

**Comment by jnothman** [comment]
Thanks @hlydecker!

**Pull request merged**